packages/jest-environment-miniflare/src/index.ts

```diff
@@ -30,12 +33,30 @@ import { installCommonGlobals } from "jest-util";
 import { MockAgent } from "undici";
 import { StackedMemoryStorageFactory } from "./storage";
 
+export class ExecutionContext {
+  [kWaitUntil]: Promise<unknown>[] = [];
+
+  passThroughOnException(): void {}
+
+  waitUntil(promise: Promise<any>): void {
+    this[kWaitUntil].push(promise);
+  }
+
+  flush<WaitUntil extends any[] = unknown[]>(): Promise<WaitUntil> {
+    return Promise.all(this[kWaitUntil]) as Promise<WaitUntil>;
```

Comment:
Apologies, I might be doing this wrong, but I get an error if I don't explain that any doesn't fall outside unknown.